pom.xml

```diff
@@ -52,13 +52,6 @@
     </developers>
 
     <properties>
-        <amps.version>8.0.2</amps.version>
-        <applinks-api.version>7.1.3</applinks-api.version>
-        <atlassian-application-api.version>1.2</atlassian-application-api.version>
-        <atlassian-crowd.version>2.10.6</atlassian-crowd.version>
-        <atlassian-mail.version>5.0.0</atlassian-mail.version>
-        <atlassian.spring.scanner.version>2.1.5</atlassian.spring.scanner.version>
-        <atlassian-user.version>3.0</atlassian-user.version>
```

Comment:
stimmt ja, der einfache check war ja in der pom.xml zu prüfen, ob keine atlassian deps mehr vorhanden sind. Ich denke das reicht, brauchst keine weiteren unit tests mehr einbauen.